.github/workflows/ci.yml

```diff
@@ -12,17 +12,20 @@ on:
     paths-ignore: ['**/*.md']
 
 jobs:
-  windows:
-    uses: ./.github/workflows/windows.yml
-    permissions:
-      contents: write
-
-  macos:
-    uses: ./.github/workflows/macos.yml
-    permissions:
-      contents: write
-
-  linux:
-    uses: ./.github/workflows/linux.yml
-    permissions:
-      contents: write
+  build:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        version: [0.11.0, '']
+      fail-fast: false
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Setup Zig
+        uses: goto-bus-stop/setup-zig@v2
+      - uses: actions/checkout@v4
+      - name: Build examples
+        if: runner.os != 'Windows'
```

Comment:
The condition is there since bulding examples on Windows is currently commented out. It wasn't planned to change current functionality in the PR just to refractor to a simpler status quo.

Reply:
If building examples on Windows works ofc it should be removed.

Reply:
We should build a static link case on windows

Dynamic links are currently not passable on windows